soccer/gameplay/plays/offense/free_goal.py

```diff
@@ -0,0 +1,50 @@
+import play
+import behavior
+import robocup
+import skills.line_kick
+import tactics.defense
+import main
+import constants
+import enum
+import role_assignment
+
+
+class FreeGoal(play.Play):
```

Comment:
It's basically just test pivot kick aimed at the correct goal so it might be better to make an actual pivot kick play to replace this. Also, that would let us avoid forgetting to switch the target for test pivot kick and have it shoot at our own goal again.